src/CaptainsLog/campaign/intel/RuinsIntelv2.java

```diff
@@ -76,7 +77,7 @@ public static boolean doesNotHaveUnexploredRuins(SectorEntityToken token) {
                 token.getMemoryWithoutUpdate().getBoolean(IGNORE_RUINS_MEM_FLAG) ||
                 !market.isPlanetConditionMarketOnly() ||
                 !hasRuins(market) ||
-                market.getSurveyLevel() != MarketAPI.SurveyLevel.FULL ||
+                // market.getSurveyLevel() != MarketAPI.SurveyLevel.FULL ||
```

Comment:
Unsure whether we want to keep some form of it, or just move over to the new one